subprojects/sfml.wrap

```diff
@@ -6,4 +6,4 @@ source_hash = 82535db9e57105d4f3a8aedabd138631defaedc593cab589c924b7d7a11ffb9d
 patch_directory = sfml
 
 [provide]
-sfml = sfml_dep
+dependency_names = sfml, sfml-all, sfml-graphics, sfml-windows, sfml-system, sfml-audio, sfml-network
```

Comment:
Typo `sfml-windows` → `sfml-window`.
```suggestion
dependency_names = sfm, sfml-all, sfml-graphics, sfml-window, sfml-system, sfml-audio, sfml-network
```
I would also remove `sfml`: the official packages don't provide a pkg-config entry for it.

Reply:
ok I wanted to keep it to keep an API continuity for those who still have dependency('sfml') in their code but I understand your point of view I take it back thank you fix by 2f86ec851962dde7f2c8d71a83d7411153501281